### PR TITLE
feat: allow opening new tab for some navigation buttons

### PR DESCRIPTION
**Pull request by pengx17**
fix AF-1010

### CODE REVIEW

**Comment by pengx17** [comment]
* **#7764** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7764?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7759** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7759?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7748** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7748?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7753** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7753?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/pages/workspace/all-page/all-page-header.tsx [hunk 1]
```diff
@@ -59,8 +59,12 @@ export const AllPageHeader = ({
               styles.headerCreateNewButton,
               !showCreateNew && styles.headerCreateNewButtonHidden
             )}
-            onCreateEdgeless={createEdgeless}
-            onCreatePage={createPage}
+            onCreateEdgeless={e =>
+              createEdgeless(e?.metaKey || e?.ctrlKey ? 'new-tab' : true)
```
Comment (forehalo) [inline packages/frontend/core/src/pages/workspace/all-page/all-page-header.tsx:63]: a todo is really needed here. such logic works but too raw.

Comment (pengx17) [reply packages/frontend/core/src/pages/workspace/all-page/all-page-header.tsx:63]: what about a `hasModifiers` check for metaKey/ctrlKey?

Comment (forehalo) [reply packages/frontend/core/src/pages/workspace/all-page/all-page-header.tsx:63]: day after again say

**Review by forehalo — APPROVED** [review]

**Pull request merged**